paddle/fluid/pybind/eager_method.cc

```diff
@@ -1940,6 +2077,32 @@ static PyObject* tensor_method_is_dist(TensorObject* self,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+PyDoc_STRVAR(tensor_is_sparse__doc__,
+             R"DOC(is_sparse($self, /)
+--
+
+Returns whether the input Tensor is SparseCooTensor or SparseCsrTensor.
+
+Input SparseCooTensor/SparseCsrTensor: returns True
+Input DenseTensor: returns False
```

Comment:
```suggestion
Input SparseCooTensor/SparseCsrTensor: returns True

Input DenseTensor: returns False
```

![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/e82017bb-2232-4e81-9a73-cec36e4782d8)
连起来了不太美观